packages/@stylexjs/babel-plugin/__tests__/legacy/stylex-transform-legacy-shorthands-test.js

```diff
@@ -462,3 +462,475 @@ describe('Legacy-shorthand-expansion resolution', () => {
     });
   });
 });
+
+describe('legacy-shorthand-expansion resolution (enableLogicalStylesPolyfill: false)', () => {
+  describe('while using RN non-standard shorthands', () => {
+    test('padding: basic shorthand', () => {
+      expect(
+        transform(
+          `
+          import stylex from 'stylex';
+          export const styles = stylex.create({
+            foo: {
+              padding: 5
+            }
+          });
+          stylex(styles.foo);
+        `,
+          { enableLogicalStylesPolyfill: false },
+        ),
+      ).toMatchInlineSnapshot(`
+        "import _inject from "@stylexjs/stylex/lib/stylex-inject";
+        var _inject2 = _inject;
+        import stylex from 'stylex';
+        _inject2(".x123j3cw{padding-top:5px}", 4000);
+        _inject2(".x1gabggj{padding-inline-end:5px}", 3000);
+        _inject2(".xs9asl8{padding-bottom:5px}", 4000);
+        _inject2(".xaso8d8{padding-inline-start:5px}", 3000);
+        export const styles = {
+          foo: {
+            kLKAdn: "x123j3cw",
+            kwRFfy: "x1gabggj",
+            kGO01o: "xs9asl8",
+            kZCmMZ: "xaso8d8",
+            $$css: true
+          }
+        };
+        "x123j3cw x1gabggj xs9asl8 xaso8d8";"
+      `);
+    });
+
+    test('margin: basic shorthand', () => {
+      expect(
+        transform(
+          `
+          import stylex from 'stylex';
+          const styles = stylex.create({
+            foo: {
+              margin: '10px 20px 30px 40px'
+            }
+          });
+          stylex(styles.foo);
+        `,
+          { enableLogicalStylesPolyfill: false },
+        ),
+      ).toMatchInlineSnapshot(`
+        "import _inject from "@stylexjs/stylex/lib/stylex-inject";
+        var _inject2 = _inject;
+        import stylex from 'stylex';
+        _inject2(".x1anpbxc{margin-top:10px}", 4000);
+        _inject2(".x3aesyq{margin-inline-end:20px}", 3000);
+        _inject2(".x4n8cb0{margin-bottom:30px}", 4000);
+        _inject2(".x11hdunq{margin-inline-start:40px}", 3000);
+        "x1anpbxc x3aesyq x4n8cb0 x11hdunq";"
+      `);
+    });
+
+    test('padding: with longhand property collisions', () => {
+      expect(
+        transform(
+          `
+          import stylex from 'stylex';
+          const styles = stylex.create({
+            foo: {
+              padding: 5,
+              paddingEnd: 10,
```

Comment:
We didn't already codemod these away? Is there a reason to do this rather than finish the code migration first?